examples/ColorSpaceInterpolations.elm

```diff
@@ -0,0 +1,231 @@
+module ColorSpaceInterpolations exposing (main)
```

Comment:
<img src="https://user-images.githubusercontent.com/69144/76228219-569ba700-6218-11ea-9dad-3d4d2f7479c7.png" />

I apologise for this not being documented anywhere clearly, but all examples are expected to fit into a 990 × 504 frame. This is the size of the iFrame they end up embedded in as well as the screenshots. This is what it will look like on the final site.

You can preview the example website here: https://5e665bbe58d21fbabc0f8a53--elm-visualization.netlify.com